script/Ciel/Day 13/13_01_01_CIEL13_4_2.txt

```diff
@@ -1800,17 +1932,17 @@ Today isn't even Christmas, so why―――
 [sha:51a62ff7960034abbd3aa1d0ceff27970bd5683c]{
 -- Page 507, Offset 34742.
 -- 先輩が俺を突き飛ばす。
-Senpai pushes me away.
+Senpai pushes me out of the way.
 }
 [sha:ff03753fd2b7c6b7fa8e20d98edbc993645654c4]{
 -- Page 507, Offset 34743.
 -- その直後―――
-Right after that―――
+As soon as she does―――
 }
 [sha:aadd294bfadc5de64abccc464becb2ca433209e7]{
 -- Page 508, Offset 34744.
 -- 鉄柵のような細長い凶器が、昆虫採集の標本のように、シエル先輩の体を貫いていた。
-A long, narrow weapon that resembles an iron fence skewers Ciel-senpai like she's an entomology specimen.
+A long, narrow weapon that looks like the spire of a wrought iron fence skewers Ciel-senpai like she's a butterfly in an insect collection.
```

Comment:
Weird comparison.

Perhaps 
>like she's a specimen in a butterfly collection.

Is the call? Both "insect collection" and "butterfly collection" seem to be used. Butterfly is great to have in here, through, since that's one of DA Noel's themes.